src/components/Settings.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+<div>
+    <el-dropdown @command="handleCommand">
+        <el-button split-button type="primary" >
+        <font-awesome-icon class="icon" icon="cog"/>
+        </el-button>
+        <el-dropdown-menu slot="dropdown">
+          <el-dropdown-item v-for="(setting, index) in settings" :key="setting.label" :command="index">
+            {{setting.label}}
+          </el-dropdown-item>
+        </el-dropdown-menu>
+    </el-dropdown>
+</div>
+</template>
+
+<script>
+import { mapActions } from 'vuex'
+
+export default {
+    name: 'Settings',
+    props: {
+      settings: Array,
+    },
+    methods: {
+      handleCommand(index) {
+        this.settings[index].action()
+      }
+    }
+}
+</script>
+
+<style scoped lang="scss">
+@import '@/styles/theme.scss';
+@import '@/styles/app.global.scss';
```

Comment:
AFAIK we don't need to import global styles. These styles are imported in the global scope to apply all components